oaknational/Oak-Web-Application
# [CUR-736] Add subject tier filtering to downloads journey

## Description

Music year: {owa_music_year}

- Added child subject/tier filtering for relevant subjects in the curriculum downloads journey

## How to test

1. Go to {owa_deployment_url}
2. Navigate to Secondary Maths - you should only see the tier filter
3. Navigate to Secondary Science (all exam boards) - you should see both filtering by child subject and tier
4. No primary programmes should have the selector

## Figma reference
https://www.figma.com/design/3cz9fb0ObFDLv1c212OtJv/Final-designs---Curriculum-squad?node-id=1903-44116&t=PBV2mNxaHpW0w1ju-0

## Checklist
